assets/js/util/index.js

```diff
@@ -690,3 +690,14 @@ export function sortObjectProperties( obj ) {
 	} );
 	return orderedData;
 }
+
+/**
+ * Gets the meta key for the given user option.
+ *
+ * @param {string} userOptionName User option name.
+ * @param {Object} _googlesitekitBase Site Kit base data (used for testing).
+ * @return {string} meta key name.
+ */
+export function getMetaKeyForUserOption( userOptionName, _googlesitekitBase = window._googlesitekitBase ) {
+	return _googlesitekitBase.blogPrefix + userOptionName;
```

Comment:
We probably need to export the value of `Context::is_network_mode()` from PHP to JS as well (would fit well into base data) and only prefix here if that is false.

Reply:
Alternatively, we could set `blogPrefix` to an empty string if `! is_network_mode`.  What do you think? Maybe then it would be more specific as a `userOptionPrefix`?

Reply:
I'd prefer to export more generic values about the "environment" under `_googlesitekitBase`. Both `blogPrefix` and `isNetworkMode` (especially the latter) could come in handy in JS in the future anyway, where-as `userOptionPrefix` IMO is too specific to be included in `_googlesitekitBase`.